gcp/tests/test_qualtrics_scheduling.py

```
@@ -0,0 +1,352 @@
+    def test_no_phone_field(self):
+        p = ConnectParticipantData(
+            response_id="R_connect_test",
+            connect_id="aabbccdd11223344aabbccdd11223344",
+            selected_date=date(2026, 6, 10),
+            timezone="US/Eastern",
+            consent_given=True,
+        )
+        assert not hasattr(p, "phone")
+
+    def test_consent_false_raises(self):
+        with pytest.raises(ValueError):
```

Ruff (0.15.15)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)